frontend/src/elm/Login/View.elm

```diff
@@ -0,0 +1,154 @@
+module Login.View exposing (view)
+
+import Html exposing (Html, div, p, text, button, input, form, label)
+import Html.Attributes exposing (id, class, name, placeholder, action, type_, checked, value, for)
+import Html.Events exposing (onClick, onInput, onCheck, onSubmit)
+
+import Login.Major.View
+import Login.Models exposing (..)
+import Login.Messages exposing (Msg(..))
+
+
+view : LoginForm -> Html Msg
+view loginForm =
+  div 
+    [ id "login" ]
+    [ div
+      [ class "tab-area" ]
+      [ tab MysnuLogin "계정으로 로그인" loginForm
+      , tab FileLogin "성적파일로 입력" loginForm
+      ]
+
+    , div
+      [ class "login-form" ]
+      [ case loginForm.loginType of
+          MysnuLogin ->
+            viewMysnuLoginForm loginForm
+            
+          FileLogin ->
+            viewFileLoginForm loginForm
+      ]
+    ]
+
+
+tab : LoginType -> String -> LoginForm -> Html Msg
+tab loginType title loginForm =
+  div
+    [ type_ "button"
+    , class (if loginForm.loginType == loginType then "tab active" else "tab")
+    , onClick (UpdateLoginType loginType)
+    ]
+    [ p [] [ text title ] ]
+
+
+viewMysnuLoginForm : LoginForm -> Html Msg
+viewMysnuLoginForm loginForm =
+  form
+    [ action "#"
+    , id "mysnu-login-form"
+    ]
+    [ usernameField loginForm.mysnuLoginForm.username
+    , passwordField loginForm.mysnuLoginForm.password
+    , checkboxField loginForm.mysnuLoginForm.useMysnuMajors
+
+    , if loginForm.mysnuLoginForm.useMysnuMajors then
+        div [] []
+      else
+        Html.map MajorMsg (Login.Major.View.view loginForm.majorForm)
+    
+    , submitButton MysnuLogin
+    ]
+
+usernameField : String -> Html Msg
+usernameField username =
+  div
+    [ class "inline" ]
+    [ label
+      [ for "username" ]
+      [ text "ID" ]
+    , input
+      [ type_ "text"
+      , id "username"
```

Comment:
I will be sure not to make conflict between IDs from the design level, so rest assured :)